src/transform/Polyfill.js

```diff
@@ -17,6 +17,14 @@ const matchesSelector = (el, selector) => {
   return false
 }
 
+/**
+ * @param {!Element} element
+ * @param {!string} selector
+ * @return {!Element[]}
+ */
+const querySelectorAll = (element, selector) =>
```

Comment:
Me too. I'm not sure what's best practice for a library such as pagelib. I was thinking if we did it, we'd want to call it the same name, `querySelectorAll()`, which would have these pros and cons as I understand it:

Pros:
- Fewer missed polyfill mistakes.
- Less typing.
- Platform differences are better abstracted.

Cons:
- Potential incompatibility with other client libraries.

I say we punt on this for now until we feel strongly one way or another.